vendor/brave-ios/Ads/BATBraveAds.mm

```diff
@@ -388,12 +388,12 @@ - (void)loadSampleBundle:(ads::OnLoadSampleBundleCallback)callback
   callback(ads::Result::SUCCESS, std::string(contents.UTF8String));
 }
 
-- (void)loadUserModelForLocale:(const std::string &)locale callback:(ads::OnLoadCallback)callback
+- (void)loadUserModelForLanguage:(const std::string &)language callback:(ads::OnLoadCallback)callback
 {
   const auto bundle = [NSBundle bundleForClass:[BATBraveAds class]];
-  const auto localeKey = [[[NSString stringWithUTF8String:locale.c_str()] substringToIndex:2] lowercaseString];
-  const auto path = [[bundle pathForResource:@"locales" ofType:nil]
-                     stringByAppendingPathComponent:[NSString stringWithFormat:@"%@/user_model.json", localeKey]];
+  const auto languageKey = [[[NSString stringWithUTF8String:language.c_str()] substringToIndex:2] lowercaseString];
+  const auto path = [[bundle pathForResource:@"user_models/languages" ofType:nil]
```

Comment:
The `user_models` folder is not copied over based on the change here: https://github.com/brave/brave-core/pull/3235/files#diff-3c34a900bdaaf0783cbfcf0e95a15c96R158, only `languages`. Possible change:

1. In BUILD.gn copy `user_models` folder instead of `user_models/languages`
2. In this change for loading said files, look in the `languages` directory instead of `user_models/langauges`

The former seems better

Reply:
Fixed

Reply:
I'll have to double check that `pathForResource:ofType:` accepts a path in the name, otherwise this may have to just be `const auto path = [[bundle pathForResource:@"user_models" ofType:nil]`and then down below `[NSString stringWithFormat:@"%@/languages/user_model.json", languageKey]`